prisma/schema.prisma

```diff
@@ -435,6 +435,7 @@ model PrismaPoolStakingGauge {
     gaugeAddress        String
     rewards             PrismaPoolStakingGaugeReward[]
     status              PrismaPoolStakingGaugeStatus    @default(ACTIVE)
+    version             Int                             @default(1)
```

Comment:
needs to be added to pool.prisma not schema.prisma

schema.prisma is generated by `yarn prisma-merge`